giftcard/src/main/java/com/adyen/checkout/giftcard/GiftCardComponentState.kt

```diff
@@ -18,8 +18,8 @@ import kotlinx.parcelize.Parcelize
  */
 @Parcelize
 class GiftCardComponentState(
```

Comment:
Should we also make this one a `data class`?

Reply:
And do you know why this one has `@Parcelize` and others don't? Should we make it consistent?

Reply:
I think it's because this is the only one we keep in the saved state. I say let's leave them like this until we need to be parcelized.